samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
Names we decided on:

Ticket/Result:
Success
NoResult
Fail

RequestHandler:
SkipHandler
HandleResponse

Challenge/JwtBearer Skips become HandleResponse as well preserving what they were called before.